web/styles/popovers.css

```diff
@@ -582,9 +582,9 @@ ul.popover-group-menu-member-list {
     .group-list-top-section,
     .stream-list-top-section {
         display: flex;
-        margin-top: 12px;
+        margin-top: 0.75em;
```

Comment:
The commit message for this one:

> This commit changes rule values from px to em for stream list and group list in user profile modal as part of a prep commit.

Is not what I'd write in a description. For one thing, you never need to say "This commit" or "as part of a prep commit" -- the placement of the commit in a series is obvious from the context. And the other text is already in the title. So I'd write something like:

> This allows the spacing to scale with the font size setting.

That explains why it's valuable to do it,